### PR TITLE
docs: Corrected all the grammatical errors in README.md

### PR DESCRIPTION
**Pull request by alienishi**
I have corrected all the grammatical errors in README.md.

### CODE REVIEW

**Comment by alienishi** [comment]
@Tienisto 
Please review my PR and if according to the guidelines please do merge it :)

File: README.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ flutter build apk
 AppBundle for Google Play
 
 ```bash
-flutter build appbundle
+flutter build app bundle
```
Comment (Tienisto) [inline README.md:136]: This command was correct, the new `flutter bulid app bundle` is incorrect

File: README.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ flutter build ipa
 ### macOS
 
 ```bash
-flutter build macos
+flutter build macOS
```
Comment (Tienisto) [inline README.md:148]: Same here, the commands are correct already. They do not follow grammatical rules

**Review by Tienisto — CHANGES_REQUESTED** [review]

**Comment by alienishi** [comment]
@Tienisto 
My sincere apologies for the mistake
I'll correct them right away

**Comment by alienishi** [comment]
@Tienisto 
Please review it once again
I have changed them back and if according to the guidelines please do merge it :)

File: README.md [hunk 3]
```diff
@@ -145,7 +145,7 @@ flutter build ipa
 ### macOS
 
 ```bash
-flutter build macos
+flutter build macoS
```
Comment (Tienisto) [inline README.md:148]: The macOS and Linux commands must stay unchanged because they are correct already

**Comment by alienishi** [comment]
@Tienisto 
Done please check once again

**Comment by Tienisto** [comment]
LGTM

**Pull request merged**